.travis.yml

```diff
@@ -6,12 +6,11 @@ cache:
       
 language: php
 php:
-    - "5.3"
-    - "5.4"
-    - "5.5"
-    - "5.6"
-    - "7.0"
-    - "hhvm"
+    - 7.1
+    - 7.2
+    - 7.3
+    - 7.4snapshot
+    - nightly
```

Comment:
I would remove nightly.

Reply:
Yes, I removed it. Also removed 7.4snapshot for it was also causing issue downloading symfony/router from packagist. 
![Screenshot from 2019-06-20 13-04-36](https://user-images.githubusercontent.com/120454/59829877-0adc6100-935c-11e9-86a2-76b0ce26776a.png)

Reply:
If we want add the `php-nightly`, it should add this in [allow_failures ](https://docs.travis-ci.com/user/build-matrix/#matching-jobs-with-allow_failures)block.

Nobody can guarantee that the `php-nightly` will be successful in every Travis CI build.

Reply:
Seems that was wrong due to some reason and 7.4 is working fine.

Reply:
if we allow failure we probably don't care it. So may be it is not needed for now ?